bin/pycbc_multi_inspiral

```diff
@@ -513,6 +528,10 @@ with ctx:
             sigma = {ifo : np.sqrt(sigmasq[ifo]) for ifo in opt.instruments}
             # Every time s_num is zero or we skip the segment, we run new 
             # template to increment the template index
+            ####### Testing values
+            wp,wc=get_weighted_antenna_patterns(Fp,Fc,sigma)
+            projection_matrix = get_projection_matrix(wp,wc)
+            LCprojection_matrix, RCprojection_matrix = circular_projection_matrix(wp, wc)
```

Comment:
We also need to account for the normal GRB search behaviour where both right and left projections are used.